.travis.yml

```diff
@@ -145,7 +145,7 @@ script:
         # Uncomment this when `coverage` runs on Trusty.
         # set -eo pipefail
         if [ "$T" = "coverage" ]; then
-             export CC="gcc-6"
+             export CC="gcc-7"
```

Comment:
When I updated the GCC version in https://github.com/curl/curl/commit/828187d86b70af3a031b529312b8027b4ece10a5, I had to change the gcov version too.

Reply:
Ah yes, I *suspected* this as well... thanks!